src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
You should have two interfaces 
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```